rust-client/Cargo.toml

```diff
@@ -11,7 +11,7 @@ env_logger = "0.9"
 either = "1.6.1"
 futures = "0.3"
 futures-timer = "3.0.0"
-libp2p = { version = "0.50.0", features = ["async-std", "dcutr", "dns", "identify", "noise", "relay", "ping", "tcp", "yamux", "macros"] }
+libp2p = { version = "0.50.0", git = "https://github.com/elenaf9/rust-libp2p", branch = "quic/draft-29", features = ["async-std", "dcutr", "dns", "identify", "noise", "relay", "ping", "quic", "tcp", "yamux", "macros"] }
```

Comment:
As discussed in a call, let's use a commit hash here.

Reply:
:+1: 63c45cf